doc/logging.md

```diff
@@ -30,125 +26,108 @@ A full log message could therefore look like this:
 This tells us that there was a debug message at *1s + 200ns*, informing us that
 the component `system.uart0` has been disabled.
 
-Log messages can be generated using the global top level logging functions from
-the `vcml` namespace. These functions use the current SystemC process name as
+Log messages can be generated using the global top level logging function-like macros from
+the `mwr` namespace. These macros use the file and line name as
```

Comment:
file name and line number?

Reply:
it doesnt, the macro confused me. corrected